tcms/testplans/tests/test_new_plan.py

```diff
@@ -70,4 +77,48 @@ def _test_plan_create_new(self, is_active):
         self.assertTrue(plan.emailing.auto_to_case_owner)
         self.assertTrue(plan.emailing.auto_to_case_default_tester)
         self.assertFalse(plan.emailing.notify_on_plan_update)
-        self.assertFalse(plan.emailing.notify_on_case_update)
\ No newline at end of file
+        self.assertFalse(plan.emailing.notify_on_case_update)
+
+
+class TestNewPlanWithoutPermission(BaseNewPlanTest):
```

Comment:
When testing permissions you need explicit tests with and without permissions (possibly where the user is not super-user).

So instead of adding several layers of abstractions use the same test class, `NewPlanViewTest`, make sure `setUpTestData()` assigns permissions to the user for the positive scenario and then in the negative scenario simply remove the permissions of the user (may need to re-login)).

Reply:
Done. No re-login was required